f2/src/forms/HowDidItStartForm.js

```diff
@@ -2,17 +2,16 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { useLingui } from '@lingui/react'
 import { Trans } from '@lingui/macro'
-import { Form, Field } from 'react-final-form'
+import { Form } from 'react-final-form'
 import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
 import { CheckboxAdapter } from '../components/checkbox'
 import { RadioAdapter } from '../components/radio'
-import { Stack, Box, Alert, AlertIcon, FormControl } from '@chakra-ui/core'
+import { Stack, Alert, AlertIcon } from '@chakra-ui/core'
 import { useStateValue } from '../utils/state'
 import { FormArrayControl } from '../components/FormArrayControl'
 import { TextArea } from '../components/text-area'
-import { P } from '../components/paragraph'
 import { TextInput } from '../components/TextInput'
-import { FormLabel } from '../components/FormLabel'
+import { Field } from '../components/Field'
```

Comment:
Our field component adapts react-final-form Field and adds label, helpertext and errorMessage